src/flaskwebgui.py

```diff
@@ -224,11 +228,15 @@ def get_browser_command(self):
         flags.extend([f"--app={self.url}"])
 
         return flags
-
+    print("AAAA")
     def start_browser(self, server_process: Union[Thread, Process]):
-        print("Command:", " ".join(self.browser_command))
-        subprocess.run(self.browser_command)
+        if self.__default_command:
+            subprocess.run(self.browser_command)
+        else:
+            self.browser_command = [self.browser_path] + [f"--app={self.url}"] + self.browser_command
+            subprocess.run(self.browser_command)

```

Comment:
In the `browser_command` of `FlaskUI` the dev should provide the full subprocess cli command (including `browser_path`). You can easily get the default `browser_path` just by importing `find_browser`.